config/locales/en.yml

```diff
@@ -2658,6 +2678,7 @@ en:
   multiple_select_option: -- multiple --
   multiple_taxa_for_taxon_change: "Multiple taxa (input for swap/merge, output for split)"
   multiple_taxon_ids_separated_by_commas: 'Multiple taxon IDs separated by commas, e.g. 123,654'
+  must_be_an_accepted_value: must be an accepted value
```

Comment:
Definitely good to translate these errors, but this translation will not be used when you do `errors.add(:iucn_relationship, :must_be_an_accepted_value)`. For that to work it needs to exist at `activerecord.errors.models.taxon_range.attributes.iucn_relationship.must_be_an_accepted_value`